src/main/java/com/autotune/analyzer/serviceObjects/CreateExperimentAPIObject.java

```diff
@@ -47,6 +47,8 @@ public class CreateExperimentAPIObject extends BaseSO {
     private RecommendationSettings recommendationSettings;
     @SerializedName(KruizeConstants.JSONKeys.DATASOURCE) //TODO: to be used in future
     private String datasource;
+    @SerializedName(KruizeConstants.JSONKeys.EXPERIMENT_TYPE) //TODO: to be used in future
```

Comment:
Create util and interface like this

`public class ExperimentTypeUtil {
    public static boolean isContainerExperiment(String experimentType) {
        return experimentType == null || experimentType.equalsIgnoreCase(AnalyzerConstants.ExperimentTypes.CONTAINER_EXPERIMENT);
    }

    public static boolean isNamespaceExperiment(String experimentType) {
        return experimentType != null && experimentType.equalsIgnoreCase(AnalyzerConstants.ExperimentTypes.NAMESPACE_EXPERIMENT);
    }
}

public interface ExperimentTypeAware {
    String getExperimentType();
}`

Reply:
and use interface with CreateExperimentAPIObject and KruizeObject like this

`public class CreateExperimentAPIObject implements ExperimentTypeAware {

public class KruizeObject implements ExperimentTypeAware 
`

Reply:
and in both object have this

`   public boolean isContainerExperiment() {
        return ExperimentTypeUtil.isContainerExperiment(experimentType);
    }

    public boolean isNamespaceExperiment() {
        return ExperimentTypeUtil.isNamespaceExperiment(experimentType);
    }`

Reply:
Added `ExperimentTypeAware` interface and `ExperimentUtil` class with static functions to verify the exp type.